test/TestProjects/Models-Cadl/Generated/FixedIntEnum.Serialization.cs

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+
+namespace ModelsInCadl
+{
+    internal static partial class FixedIntEnumExtensions
+    {
+        public static int ToSerialString(this FixedIntEnum value) => value switch
```

Comment:
Another thing to improve is the name of the serialization method. It's called `ToSerialString` which cannot cover other types we plan to support. Better to rename it to `ToSerializationValue`

Reply:
I changed the method name to `ToSerial{Type}`, so that for existing enum types the method name is not changed.